### PR TITLE
Always send Content-Length header as string.

### PR DESCRIPTION
**Pull request by jneves**
This will solve issue https://github.com/freakboy3742/pyxero/issues/173

---
This change is [Reviewable](https://reviewable.io/reviews/freakboy3742/pyxero/174)

### CODE REVIEW

**Comment by romgar** [comment]
Thanks @jneves for the fix!! Can you please update the `ChangeLog.rst` file with your change?

File: ChangeLog.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ master (unreleased)
     - GH-151: Add delete method in BaseManager (thanks Craig Handley).
 
 *Bugfix:*
-
+    - GH-173: Send Content-Legth as string for working attachment uploads (thanks João Neves).
```
Comment (romgar) [inline ChangeLog.rst]: Change `Legth` to `Length` please.

**Comment by romgar** [comment]
Can you please resolve the conflict on `ChangeLog.rst` please?

**Comment by jneves** [comment]
@romgar Done!

**Comment by romgar** [comment]
Thanks @jneves !!

**Pull request merged**